### DISCUSSION TITLE
Feature : Add Dark Mode Button

### DISCUSSION THREAD

**Discussion by mohit-100** [rewindui/rewindui#95] [Ideas]
##Description:
We would like to propose the addition of a Dark Mode feature to improve the user experience of our application. Dark Mode is becoming increasingly popular among users, as it reduces eye strain, conserves battery life on OLED screens, and provides a sleek, modern look.

##Feature Request Details:

##Objective: Implement Dark Mode in our application.
Benefits:
Improved user experience, especially in low-light environments.
Staying competitive with other applications that offer Dark Mode

**Comment by NickDunas** [comment]
Dark mode is on the roadmap and will be implemented at some point!

**Comment by mohit-100** [comment]
If you have any updates or timelines regarding the implementation of Dark Mode, please keep us informed

**Reply by NickDunas** [reply]
I don't have an exact ETA. The plan is to finish with the test suite of the library first, so that the library can be considered stable. Afterwards I plan to spend some time trying to spread the word, as the library is not that popular yet. Next step will be dark mode!

It all depends on my free time though. Feel free to follow me on X if you want to get informed! :)
